dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigValidationResult.java

```diff
@@ -44,7 +44,15 @@ public static enum ValidationFailedReason {
 
         DUPLICATE_KEY,
 
-        UNSUPPORTED_PROTOCOL;
+        UNSUPPORTED_PROTOCOL,
+
+        MISSING_PUBKEY_NAME,
+
+        INVALID_VERIFY_OPTION,
```

Comment:
is `INVALID_VERIFY_OPTION` used? I don't see it in the case below, it might be elsewhere in the code.

Reply:
Yes, it's used in `PropertiesUtils.java` inside `com.ibm.ws.install.featureUtility`